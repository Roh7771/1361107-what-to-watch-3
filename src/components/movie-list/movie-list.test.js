import React from "react";
import renderer from "react-test-renderer";
import MovieList from "./movie-list";

const mock = {
  filmsList: [
    {
      title: `Any Cool Film`,
      imgSrc: `Some path`,
      id: 5
    },
    {
      title: `Another Cool Film`,
      imgSrc: `Some path`,
      id: 9
    },
  ]
};

it(`<Main /> should render correctly`, () => {
  const tree = renderer
    .create(<MovieList filmsList={mock.filmsList} onTitleButtonClick={() => {}}/>)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
