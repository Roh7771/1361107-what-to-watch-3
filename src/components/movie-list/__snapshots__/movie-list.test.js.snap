// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Main /> should render correctly 1`] = `
Array [
  <article
    className="small-movie-card catalog__movies-card"
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
  >
    <div
      className="small-movie-card__image"
    >
      <img
        alt=""
        height="175"
        src="Some path"
        width="280"
      />
    </div>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="movie-page.html"
        onClick={[Function]}
      >
        Any Cool Film
      </a>
    </h3>
  </article>,
  <article
    className="small-movie-card catalog__movies-card"
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
  >
    <div
      className="small-movie-card__image"
    >
      <img
        alt=""
        height="175"
        src="Some path"
        width="280"
      />
    </div>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="movie-page.html"
        onClick={[Function]}
      >
        Another Cool Film
      </a>
    </h3>
  </article>,
]
`;
